SPECS/xcp-networkd.spec

```diff
@@ -23,15 +23,12 @@ BuildRequires:  ocaml-xen-api-client-devel
 BuildRequires:  ocaml-netlink-devel
 BuildRequires:  libffi-devel
 BuildRequires:  ocaml-bisect-ppx-devel
-BuildRequires:  ocaml-systemd-devel
-BuildRequires:  systemd-devel
+BuildRequires:  ocaml-systemd
```

Comment:
You may still need `BuildRequires:  systemd`... let's see if it works on your branch...